extras/static_code_analysis.sh

```diff
@@ -20,8 +20,8 @@
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
 
-# This shell script is part of the NEST Travis CI build and test environment.
-# It performs the static code analysis and is invoked by 'travis_build.sh'.
+# This shell script is part of the NEST Github Actions CI build and test environment.
+# It performs the static code analysis and is invoked by 'gha_build.sh'.
```

Comment:
I suggest to rename `extras/gha_build.sh` to `ci_build.sh` and change all references to it accordingly.

```suggestion
# It performs the static code analysis and is invoked by 'ci_build.sh'.
```

Reply:
Done

Reply:
Now, the script is called just `ci.sh`, which I find a bit too brief. Can you please make it `ci_build.sh`? Or is there a reason for your shorter choice?

Reply:
I somehow read your comment wrongly and assumed it as `ci.sh`. Sorry, my bad. I have fixed this correctly now as `ci_build.sh`.